less/styles.less

```diff
@@ -56,10 +56,10 @@ body {
   background-size: cover;
   height: 45rem;
   color: white;
+  display: flex;
+  flex-direction: column;
+  justify-content:center;
```

Comment:
I'm not sure how much we care about formatting here if it's a thing then spacing otherwise ignore me